py/dml/dmlparse.py

```diff
@@ -2674,15 +2685,27 @@ def ident_enforce_not_discard_ref(site, ident):
                                          'ASYNC', 'AWAIT', 'WITH']
 
 @prod_dml12
-@lex.TOKEN(ident_rule(reserved_words_12))
+@lex.TOKEN(ident_rule('ident', reserved_words_12))
 def reserved(t):
     raise ESYNTAX(site(t, 1), str(t[1]), "reserved word")
 
 @prod_dml14
-@lex.TOKEN(ident_rule(reserved_words_14))
+@lex.TOKEN(ident_rule('ident', reserved_words_14))
 def reserved(t):
     raise ESYNTAX(site(t, 1), str(t[1]), "reserved word")
 
+@prod_dml14
+@lex.TOKEN(ident_rule('ident_or_underscore', reserved_words_14))
+def reserved_(t):
+    raise ESYNTAX(site(t, 1), str(t[1]), "reserved word")
+
+@prod_dml14
+@lex.TOKEN(ident_rule('ident', ['_']))
```

Comment:
just do
```
ident : _
```
also in my WIP I called the token `DISCARDREF`.

Reply:
`_` follows the convention that identifier-formed tokens use that as their name.

Reply:
And thanks for spotting my stupid copy-paste